mmcls/datasets/transforms/formatting.py

```diff
@@ -213,55 +215,70 @@ def __repr__(self):
             f'(keys={self.keys}, order={self.order})'
 
 
-@TRANSFORMS.register_module()
-class ToPIL(BaseTransform):
+@TRANSFORMS.register_module(('NumpyToPIL', 'ToPIL'))
+class NumpyToPIL(BaseTransform):
     """Convert the image from OpenCV format to :obj:`PIL.Image.Image`.
 
     **Required Keys:**
 
-    - img
+    - ``*img**``
 
     **Modified Keys:**
 
-    - img
+    - ``*img**``
+
+    Args:
+        to_rgb (bool): Whether to convert img to rgb. Defaults to False.
     """
 
-    def transform(self, results):
+    def __init__(self, to_rgb: bool = False) -> None:
+        self.to_rgb = to_rgb
+
+    def transform(self, results: Dict) -> Dict:
```

Comment:
```suggestion
    def transform(self, results: dict) -> dict:
```
Please also check other parts